packages/framework/attributor/src/mixinAttributor.ts

```diff
@@ -134,8 +134,21 @@ export const mixinAttributor = (Base: typeof ContainerRuntime = ContainerRuntime
 			const shouldTrackAttribution = mc.config.getBoolean(enableOnNewFileKey) ?? false;
 			if (shouldTrackAttribution) {
 				(context.options.attribution ??= {}).track = true;
+				// Add telemetry to track if the storage of attribution data was enabled in configuration or not
+				mc.logger.send({
+					category: "generic",
+					eventName: "mixinAttributor_Load",
+					storageOfAttributionDataEnabled: true,
+				});
 			}
 
+			// Add telemetry logging to track using a mixed in attributor or not
+			mc.logger.send({
+				category: "generic",
+				eventName: "mixinAttributor_Load",
```

Comment:
In general, if you find yourself prefixing a lot of event names with some string, you should consider using `ChildLogger`. Once other feedback in this review is addressed, I'm not sure how necessary this will be (I expect we'll only have 1 or 2 events, which is more ok to do in a one-off fashion)

Reply:
eh, I think it's not unreasonable to expect we'll have more events in this area in the future. I'd create an `Attributor` namespace with the parent logger and put all events in that namespace. Then your eventname will look something like `...ContainerRuntime:Attributor:initialize`